docs/source/changelog.md

```diff
@@ -12,6 +12,266 @@ and as we merge [breaking changes in pull
 requests](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pulls?q=is%3Apr+is%3Aclosed+label%3Abreaking),
 this list should be updated.
 
+## 2.0
+
+### 2.0.0
+
+#### Highlights
+
+- **TODO**
+
+  TODO
+
+#### Security: breaking change to `*.networkPolicy.egress`
+
+If you have configured any of:
+
+- `hub.networkPolicy.egress`
+- `proxy.chp.networkPolicy.egress`
+- `proxy.traefik.networkPolicy.egress`
+- `singleuser.networkPolicy.egress`
+  you must review your configuration as additional default egress routes have been added.
+  Previously `...networkPolicy.egress` controlled all egress but a new properties `...networkPolicy.egressAllowRules` add additional egress rules by default.
+
+If you have configured `...networkPolicy.egress` for `hub`, `proxy.chp`,
+`proxy.traefik` or `singleuser` to restrict the permissions to establish
+outbound network connections, then this upgrade is likely to _escalate those
+permissions unless you revise your configuration_. The new configuration
+`...networkPolicy.egressAllowRules` are by default granting most of the egress
+permissions previously granted by default via the `...networkPolicy.egress`
+configuration, and `...networkPolicy.egress` are now by default not providing
+any permissions.
+
+If you for example had overridden the previously very permissive default value
+of `singleuser.networkPolicy.egress` to be less permissive, you should consider
+disabling all `singleuser.networkPolicy.egressAllowRules` like this
+to not risk escalating the permissions.
+
+```yaml
+singleuser:
+  networkPolicy:
+    egressAllowRules:
+      cloudMetadataServer: false
+      dnsPortsPrivateIPs: false
+      nonPrivateIPs: false
+      privateIPs: false
+```
+
+For more details, see the documentation on [Kubernetes Network Policies](netpol)
+and the configuration reference entries under
+[`...networkPolicy.egress`](schema_hub.networkPolicy.egress) and
+[`...networkPolicy.egressAllowRules`](schema_hub.networkPolicy.egressAllowRules).
+
+#### Upgrade instructions
+
+1. Adjust your configuration based on the security notice above if it applies to
+   your deployment.
+2. Prepare a user image with a modern version of JupyterHub installed. Install
+   `jupyterhub>=2.3.1` via pip or `jupyterhub-base>=2.3.1` via conda/mamba in
+   the image. Configure to use this new image, which is typically done via
+   `singleuser.image` or as part of `singleuser.profileList`.
+3. Read and adjust your configuration to the breaking changes below.
+
+#### Breaking changes
+
+These breaking changes have been made relative to the 1.\* series of Z2JH releases.
+
```

Comment:
```suggestion

##### JupyterHub 2 and related hub components
Z2JH 2.0.0 upgrades JupyterHub to the 2.\* series, and also upgrades all hub components.
If you are using any custom JupyterHub services, addons, API integrations, or extra configuration, you should review the breaking changes in the
[JupyterHub 2.0.0 changelog](https://github.com/jupyterhub/jupyterhub/blob/2.3.1/docs/source/changelog.md#200).

JupyterHub 2 uses an updated database schema.
Z2JH 2.0.0 automatically handles the upgrade, but it will not be possible to downgrade to older releases after this.

JupyterHub 2 adds RBAC for managing permissions in JupyterHub.
The old permissions model of admin/non-admin still works but you should use [RBAC to assign the required privileges to users or services in future](https://jupyterhub.readthedocs.io/en/stable/rbac/index.html)

See 
`TODO: link to Notable dependencies updated`
for more information on other upgraded hub components.

##### JupyterLab and Jupyter Server
The default singleuser server is [JupyterLab](https://jupyterlab.readthedocs.io/), running on [Jupyter server](https://jupyter-server.readthedocs.io/en/latest/).
To switch back to Jupyter Notebook either configure/rebuild your singleuser image to default to notebook, or see [the documentation on user interfaces](user-interfaces)

```
The anchors in the JupyterHub changelog on RTD aren't constant across versions, e.g. `https://jupyterhub.readthedocs.io/en/stable/changelog.html#id41`, so I've linked to the GitHub changelog instead.